### ISSUE TITLE
[FavoriteMedia] the plugin cannot send the link( 

### ISSUE THREAD

**Issue by XxvaleraxX** [Dastan21/BDAddons#72]
deja vu?
I hope it will be fixed soon, this is one of my favorite plugins.

**Comment by Masteroduo** [comment]
same here, when I click on the media nothing happens.

**Comment by Debuggerz1** [comment]
I checked the console when clicking on saved media, and there was an error of some sort which made no sense but it led me to this piece of code: 

line:885
```
if (!shiftPressed) {
    ComponentDispatch.dispatchToLastSubscribed("INSERT_TEXT", {
        content: this.props.url
    });
    const textarea = document.querySelector(`.${classes.textarea.textAreaSlate}`);
    if (textarea ? .firstChild ? .className.includes("placeholder")) setTimeout(() => textarea ? .firstChild ? .dispatchEvent(new KeyboardEvent("keydown", {
        key: "Enter",
        code: "Enter",
        which: 13,
        keyCode: 13,
        bubbles: true
    })), 0);
    else EPS.toggleExpressionPicker(this.props.type, EPSConstants)
} else {
    WebpackModules.getByProps("sendMessage").sendMessage(SelectedChannelStore.getChannelId(), {
        content: this.props.url,
        validNonShortcutEmojis: []
    });
}

```
so i tried to hold shift then click the image and then it did send directly wihtout pasting the link in the text box.

I'm relatively new to all this so i'm not sure what's really going on but i'm guessing it normally pasts the link in the text box and when you hold shift it sends directly? 

so all i did is change the above code to this:

```
WebpackModules.getByProps("sendMessage").sendMessage(SelectedChannelStore.getChannelId(), {
    content: this.props.url,
    validNonShortcutEmojis: []
});
```
Basically removed the shift key check and the alternate piece of code since it's broken.

Still there might be come complications to this, since I'm not fully aware of how BD internals work.
